security-insights.yml

```
@@ -0,0 +1,64 @@
+header:
+  schema-version: 2.0.0
+  last-updated: '2025-04-10'
+  last-reviewed: '2025-04-10'
+  url: https://github.com/ossf/security-insights-spec/releases/download/v2.0.0/schema.cue
+
+project:
+  name: Backstage
+  homepage: https://backstage.io
+  roadmap: https://backstage.io/docs/overview/roadmap/
+  administrators:
+    - name: Patrik Oldsberg
+      affiliation: Spotify
+      email: poldsberg@gmail.com
+      primary: true
+    - name: Fredrik Adelöw
+      affiliation: Spotify
+      email: freben@spotify.com
+    - name: Ben Lambert
+      affiliation: Spotify
+      email: ben@blam.sh
+  documentation:
+    quickstart-guide: https://backstage.io/docs/getting-started/quickstart
```

```suggestion
    quickstart-guide: https://backstage.io/docs/getting-started
```

Probably this one instead, current link doesn't work